.github/workflows/redocly.yml

```diff
@@ -17,6 +17,14 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 16
+      - name: Setup go-swagger
+        uses: minchao/setup-go-swagger@v1
+        with:
+          version: v0.26.1
+      - name: swagger-spec
+        run: swagger generate spec -o ./server/helpers/swagger.yaml --scan-models
```

Comment:
I wonder if we need that part ` swagger generate spec -o ./server/helpers/swagger.yaml --scan-models` , bcs the redocs docs is built based the `_data/swagger.yml`

Reply:
@Freedisch I don't think we need it, however I've added it for consistency sake